knack/arguments.py

```diff
@@ -12,9 +12,16 @@
 
 
 class CLIArgumentType(object):
+
     REMOVE = '---REMOVE---'
 
     def __init__(self, overrides=None, **kwargs):
+        """A base CLI Argument Type that can be applied to multiple command arguments
+
+        :param overrides: The base argument that you are overriding
+        :type overrides: knack.arguments.CLIArgumentType
+        :param kwargs: Kwargs for an argument (Argparse keywords can be used or you can add your own)
```

Comment:
There are kwargs that Knack supports beyond the argparse keywords (validator, for example). These should be called out here, especially since everything else points to this (however, I don't think it should point here).

Reply:
done